controllers/vehicle.go

```diff
@@ -0,0 +1,71 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/edgarduran/Go-Dispatch-Bootcamp/usecase"
+	"github.com/gorilla/mux"
+)
```

Comment:
It's not a written rule, but in Go, we try to have our imported packages separated into three main groups:
- Standard library packages
- Local packages
- 3rd party packages

This is the order that IMHO suits better for me, but be free to use any order you want, just have in mind separating the packages here and in all other occurrences in your project.

```suggestion
import (
	"encoding/json"
	"fmt"
	"net/http"
	"strconv"

	"github.com/edgarduran/Go-Dispatch-Bootcamp/usecase"
	
	"github.com/gorilla/mux"
)
```